src/components/DocParse.js

```diff
@@ -6,14 +6,29 @@ import css from '../css/main.module.less';
 let linkReference;
 
 function parseCodeBlock (child, index) {
+	let codeText = '';
+	let temp = '';
 	const lang = child.lang || 'html';	// HTML formatting works better on JSX than JavaScript does
-	return <Code type={lang} key={index}>{child.value}</Code>;
+	if (child.children.length === 1) {
+		return <Code type={lang} key={index}>{child.children[0].value}</Code>;
+	} else {
+		child.children.forEach((elem) => {
+			if(elem.type === 'text'){
+				temp = elem.value;
+			} else {
+				temp = parseChildren(elem)
+			}
+			codeText += temp;
+		});
+		const text = codeText.replace(/\,/g,'');
+		return <Code type={lang} key={index}>{text}</Code>;
+	}
 }
 
 function parseLink (child, index) {
 	let title = child.children[0].value;
 	const linkText = child.children[0].text || linkReference || title;
-	const url = child.url;
+	const url = (child.properties && child.properties.href);
```

Comment:
How about using optional chaining operator?
```suggestion
	const url = child.properties?.href;
```